### PR TITLE
docs: Remove duplicate Azure Identity SDK content

### PR DESCRIPTION
**Pull request by scottaddie**
Addresses https://github.com/Azure/azure-sdk-for-cpp/pull/4894#issuecomment-1684195033

### CODE REVIEW

File: docs/book/src/topics/language-specific-examples/azure-identity-sdk.md [hunk 1]
```diff
@@ -1,33 +1,3 @@
...
-| --------------------- | --------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
-| Go                    | [azure-sdk-for-go](https://github.com/Azure/azure-sdk-for-go)         | [Link](https://github.com/Azure/azure-workload-identity/tree/main/examples/azure-identity/go)     |
-| Python                | [azure-sdk-for-python](https://github.com/Azure/azure-sdk-for-python) | [Link](https://github.com/Azure/azure-workload-identity/tree/main/examples/azure-identity/python) |
-| JavaScript/TypeScript | [azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js)         | [Link](https://github.com/Azure/azure-workload-identity/tree/main/examples/azure-identity/node)   |
-| C#                    | [azure-sdk-for-net](https://github.com/Azure/azure-sdk-for-net)       | [Link](https://github.com/Azure/azure-workload-identity/tree/main/examples/azure-identity/dotnet) |
-| Java                  | [azure-sdk-for-java](https://github.com/Azure/azure-sdk-for-java)     | [Link](https://github.com/Azure/azure-workload-identity/tree/main/examples/azure-identity/java)   |
```
Comment (aramase) [inline docs/book/src/topics/language-specific-examples/azure-identity-sdk.md:33]: IMO the examples section shouldn't be removed from this doc as it refers to examples in this GitHub repo

**Review by aramase — CHANGES_REQUESTED** [review]
@scottaddie Thank you for the PR!

I'm fine with the initial sections of the doc referring to the AKS doc but the examples section should still exist here. Otherwise users will be redirected again to this repo where the examples exist.

File: docs/book/src/topics/language-specific-examples/azure-identity-sdk.md (hunk 1 above)
Comment (aramase) [reply docs/book/src/topics/language-specific-examples/azure-identity-sdk.md:33]: > What if we deleted the code samples from this repo and embedded them in the conceptual doc instead? An easy way to do that is language-specific tabs. I'll even volunteer to open that docs PR. Let me know your thoughts.

I like this idea! I can help out with this change.

Comment (scottaddie) [reply docs/book/src/topics/language-specific-examples/azure-identity-sdk.md:33]: The doc update is live at https://learn.microsoft.com/azure/aks/workload-identity-overview?tabs=dotnet#azure-identity-client-libraries.

File: examples/examples.sln [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿
```
Comment (aramase) [inline examples/examples.sln:1]: @scottaddie I believe this was accidentally added? Could you remove this file?

**Review by aramase — CHANGES_REQUESTED** [review]

**Pull request merged**